web-api/src/database-types.ts

```diff
@@ -0,0 +1,48 @@
+import { ColumnType, Insertable, Selectable, Updateable } from 'kysely';
+
+export interface Database {
+  message: MessageTable;
+  case: CaseTable;
+}
+
+export interface MessageTable {
+  attachments?: ColumnType<{ documentId: string }[], string, string>;
+  caseStatus: string;
+  caseTitle: string;
+  completedAt?: string;
+  completedBy?: string;
+  completedBySection?: string;
+  completedByUserId?: string;
+  completedMessage?: string;
+  createdAt: string;
+  docketNumber: string;
+  from: string;
+  fromSection: string;
+  fromUserId: string;
+  isCompleted: boolean;
+  isRead: boolean;
+  isRepliedTo: boolean;
+  leadDocketNumber?: string;
+  message: string;
+  messageId: string;
+  parentMessageId: string;
+  subject: string;
+  to: string;
+  toSection: string;
+  toUserId: string;
+}
+
+export type Message = Selectable<MessageTable>;
```

Comment:
Just to make sure these types do not conflict with our Message.ts Entity and we do not accidentally importing the wrong `Message` type could we rename these to be something a little different? Maybe MessageKysley or MessageKy or something differnt.